tensorflow_similarity/augmenters/augmentation_utils/cropping.py

```diff
@@ -181,26 +181,80 @@ def crop_and_resize(
         [image], [height, width], method=tf.image.ResizeMethod.BICUBIC
     )[0]
     return img
-
+ 
+def crop_random_resized_crop(
+  image: Tensor, 
+  height: int, 
+  width: int, 
+  min_cropsize_multiplier: int = 0.75,
+  max_cropsize_multiplier: int = 1
+):
+  rand_width = tf.random.uniform(
+      shape=[],
+      minval=int(min_cropsize_multiplier * width),
+      maxval=int(max_cropsize_multiplier * width),
+      dtype=tf.int32,
+  )
+ 
+  # rand_height = tf.random.uniform(
+  #     shape=[],
+  #     minval=int(min_cropsize_multiplier * height),
+  #     maxval=int(max_cropsize_multiplier * height),
+  #     dtype=tf.int32,
+  # )
+ 
+  crop = tf.image.random_crop(image, (rand_width, rand_width, 3))
```

Comment:
We should add a doc string explaining that we always take a square crop but resize to height x width. This may stretch and distort the cropped image.

Reply:
Okay, i think i've added it